portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,19 +38,51 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  ArrayList<String> list = new ArrayList<>();    
+  String commentsJson = new String();
```

Comment:
Variables should be declared at their narrowest possible scope; since this is only used in doGet, it should be declared in doGet.